src/TestUtils/src/DeviceTests.Runners/TestUtils.DeviceTests.Runners.csproj

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\Compatibility\Core\src\Compatibility.csproj" />
+    <ProjectReference Include="..\..\..\Controls\Maps\src\Controls.Maps.csproj" />
```

Comment:
Not sure why i need to add this to fix the build.

Reply:
Maybe because of this?
https://github.com/dotnet/maui/pull/7886/files#diff-3370ff94460288159ed7ef80e5541eb2b5a09502e2fd6ec01ae451c7e6afe9e6R6